rtlcss not working at runtime — Node.js not available in PATH

## Description

The `rtlcss` command installed via `pnpm install -g rtlcss` in the Dockerfile does not work at runtime.

## Steps to Reproduce

```bash
docker run mintsystem/odoo:19.0
docker exec odoo rtlcss --version
```

**Expected:** Version number printed
**Actual:**
```
ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/opt/odoo".
```

## Root Cause

Two issues:

1. **No Node.js runtime at runtime.** `pnpm` bundles its own Node.js under `/root/.local/share/pnpm`, but it's not discoverable in `$PATH` when the container runs. Running `which node` returns nothing.

2. **The pnpm shim doesn't work outside a project directory.** The file at `/root/.local/share/pnpm/rtlcss` is a shell shim that invokes pnpm, which expects a `package.json` in the current working directory. Since the container's workdir is `/opt/odoo` (no `package.json`), it fails regardless of directory.

## Relevant Dockerfile lines

```dockerfile
# Install pnpm
RUN curl -fsSL https://get.pnpm.io/install.sh | SHELL=/bin/bash bash -

# Add pnpm to PATH
ENV PNPM_HOME="/root/.local/share/pnpm"
ENV PATH="$PNPM_HOME:$PATH"

# Install Node.js packages
RUN pnpm install -g rtlcss
```

## Current Workaround

Install Node.js and rtlcss via apt/npm at container startup, and remove the broken pnpm shim:

```bash
apt-get update && apt-get install -y --no-install-recommends nodejs npm
npm install -g rtlcss
rm -f /root/.local/share/pnpm/rtlcss
```

## Suggested Fix

Replace pnpm with apt-installed nodejs + npm for rtlcss installation. See linked PR.

--- a/images/odoo/Dockerfile
+++ b/images/odoo/Dockerfile
@@ -129,15 +129,11 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ bookworm-pgdg main' > /et
     && rm -f /etc/apt/sources.list.d/pgdg.list \
     && rm -rf /var/lib/apt/lists/*
 
-# Install pnpm
-RUN curl -fsSL https://get.pnpm.io/install.sh | SHELL=/bin/bash bash -
-
-# Add pnpm to PATH
-ENV PNPM_HOME="/root/.local/share/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-
-# Install Node.js packages
-RUN pnpm install -g rtlcss
+# Install Node.js and rtlcss
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends nodejs npm && \
+    npm install -g rtlcss && \
+    rm -rf /var/lib/apt/lists/*
 
 # Install uv for Python package management
 COPY --from=ghcr.io/astral-sh/uv:0.7.21 /uv /uvx /bin/

--- a/images/odoo/Dockerfile
+++ b/images/odoo/Dockerfile
@@ -129,11 +129,15 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ bookworm-pgdg main' > /et
     && rm -f /etc/apt/sources.list.d/pgdg.list \
     && rm -rf /var/lib/apt/lists/*
 
-# Install Node.js and rtlcss
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends nodejs npm && \
-    npm install -g rtlcss && \
-    rm -rf /var/lib/apt/lists/*
+# Install pnpm
+RUN curl -fsSL https://get.pnpm.io/install.sh | SHELL=/bin/bash bash -
+
+# Add pnpm to PATH
+ENV PNPM_HOME="/root/.local/share/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+
+# Install Node.js via pnpm and install rtlcss globally
+RUN pnpm env use --global lts && pnpm install -g rtlcss
 
 # Install uv for Python package management
 COPY --from=ghcr.io/astral-sh/uv:0.7.21 /uv /uvx /bin/

